create-mobile-app.md

```diff
@@ -23,10 +30,23 @@ The simulator starts automatically and loads your app:
 
 <img height="600px;" src="img/mobile-simulator.png" />
 
-## Build your app binaries
+## Run on devices
+
+Wakanda Studio automatically detects supported device connected to the USB port.
```

Comment:
There are some pre-requisites that should be clear:

- For Android the device should be in "usb debugging" mode.
![view image](http://www.androidcentral.com/sites/androidcentral.com/files/postimages/684/usb-debugging.png)

- For iOS you need to register a Apple Developer account and setup the project with your certificates. This is possible opening the Xcode project you find in `platforms/ios`